plugins/ingest/src/main/java/org/elasticsearch/plugin/ingest/PipelineExecutionService.java

```diff
@@ -43,7 +43,7 @@ public PipelineExecutionService(PipelineStore store, ThreadPool threadPool) {
     public void execute(Data data, String pipelineId, Listener listener) {
         Pipeline pipeline = store.get(pipelineId);
         if (pipeline == null) {
-            listener.failed(new IllegalArgumentException(LoggerMessageFormat.format("pipeline with id [{}] does not exist", pipelineId)));
+            listener.failed(new IllegalArgumentException(LoggerMessageFormat.format("pipeline with id [{}] does not exist", new Object[]{pipelineId})));
```

Comment:
at runtime the wrong method was used... and there for the variable placeholder didn't work and the variable was appended at the end of the log line

By using object[] the right format method gets used.